packages/components/src/SidePanel/SidePanel.component.js

```diff
@@ -49,13 +49,32 @@ function getActionId(id, action) {
  />
  *
  */
-function SidePanel({ id, selected, onSelect, actions, docked, onToggleDock, t, renderers }) {
+function SidePanel({
+	id,
+	selected,
+	onSelect,
+	actions,
+	docked,
+	reverse,
+	large,
+	onToggleDock,
+	t,
+	renderers,
+}) {
 	const dockedCSS = { [theme.docked]: docked };
-	const navCSS = classNames(theme['tc-side-panel'], dockedCSS, 'tc-side-panel');
+	const largeCSS = { [theme.large]: large };
+	const navCSS = classNames(theme['tc-side-panel'], 'tc-side-panel', dockedCSS, largeCSS);
 	const listCSS = classNames(
-		'nav nav-pills nav-inverse nav-stacked',
-		'tc-side-panel-list',
+		'nav',
+		'nav-pills',
+		theme['nav-pills'],
+		'nav-stacked',
+		theme['nav-stacked'],
 		theme['action-list'],
+		'tc-side-panel-list',
+		theme['tc-side-panel-list'],
+		{ 'nav-inverse': !reverse },
+		{ [`${theme['nav-inverse']}`]: !reverse },
```

Comment:
I don't understand why you're using template string here instead of just doing something like that: 

```
	{
		'nav-inverse': !reverse,
		[theme['nav-inverse']]: !reverse,
	},
```

Am I missing something ?

Reply:
You're not missing something 👍